### DISCUSSION TITLE
Emgu CV Mac OSX runtime NuGet package

### DISCUSSION THREAD

**Discussion by bitterman4020** [emgucv/emgucv#425] [Q&A] [answered]
How come there is no publicly accessible NuGet package on nuget.org for Emgu CV Mac OSX runtime, something like "Emgu.CV.runtime.macos"...? There is of course something there only for Windows, Raspbian and Ubuntu. The repo seems to have nuspec files for that and other similarly not available NuGet package.

I actually just require the "cvextern.dll" compiled for Mac OSX, any chance I can get to that with just referencing some NuGet package, without adding any project reference?

**Comment by emgucv** [comment, marked as answer]
Emgu.CV.runtime.macos.4.5.1.4349.nupkg is available under the commercial license. You can email support@emgu.com to request an evaluation.
